<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>volumeClass</class>
 <widget class="QWidget" name="volumeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>94</width>
    <height>235</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>94</width>
    <height>235</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>94</width>
    <height>235</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>volume</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#volumeClass
{
	
	background-color: rgba(255, 255, 255, 0);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="baseLink" native="true">
     <property name="styleSheet">
      <string notr="true">#baseLink
{
	
	
	background-color: rgba(255, 255, 255, 0);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="body" native="true">
        <property name="styleSheet">
         <string notr="true">#body
{
	border-radius: 20px;
	border: 1px solid #000;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="volumePregress" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>150</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#volumePregress
{
	background-color: #31353E;
}</string>
           </property>
           <widget class="QWidget" name="pregressTarget" native="true">
            <property name="geometry">
             <rect>
              <x>45</x>
              <y>5</y>
              <width>4</width>
              <height>140</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>4</width>
              <height>140</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>4</width>
              <height>140</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
           </widget>
           <widget class="QWidget" name="pregressCurrent" native="true">
            <property name="geometry">
             <rect>
              <x>44</x>
              <y>75</y>
              <width>6</width>
              <height>70</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>6</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>6</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(85, 85, 127);</string>
            </property>
           </widget>
           <widget class="QWidget" name="volumeVertex" native="true">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>68</y>
              <width>14</width>
              <height>14</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(85, 85, 127);
border-radius: 7px;
border: 0px solid #000;</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="volumeValue" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#volumeValue
{
	background-color: #21252E;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="valumeLabel">
              <property name="styleSheet">
               <string notr="true">color: #1195D9;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="mute" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #21252E;</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="volumeButton">
              <property name="minimumSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#volumeButton
{
image: url(:/barVolume.png);
border: none;
}

#volumeButton::hover
{
	background-color: rgb(0, 85, 255);
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="bottom" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TexasPlayer.qrc"/>
 </resources>
 <connections/>
</ui>
